### PR TITLE
Add to-svg command and adjust to-ruby to the new Victor DSL

### LINKED ISSUE #6: Comments are converted to the word `comment`
### Reproduction code

Use this `sample.svg`

```svg
<svg>
  <!-- this comment is converted to the word 'comment' -->
  <rect x="10" y="10" width="120" height="80"/>
</svg>
```

Run this:

```shell
$ victor to-ruby sample.svg
```

Get this output:

```ruby
require "victor"

svg = Victor::SVG.new
svg.build do
  comment   # <-- this should not be here
  rect x: "10", y: "10", width: "120", height: "80"
end

svg.save "generated"
```

### PR DESCRIPTION
**Pull request by DannyBen**
**Note: This branch currently uses the Victor prereleased gem version 0.3.0.rc1** - update before releasing.

This is the same as #5, with these changes:

- Add `to-svg` command with support
- Update `to-ruby` command to support templates, in line with the new [Victor DSL](https://github.com/dannyben/victor#dsl-syntax)
- Fix added comments when generating Ruby (closes #6)

With this PR, ruby files can now look like this:

```ruby
# example.rb
setup width: 140, height: 100

build do
  circle cx: 50, cy: 50, r: 30, fill: "yellow"
end
```

### CODE REVIEW

**Comment by DannyBen** [comment]
Ok - I moved the DSL to the upstream.
So now, people using plain Victor can do:

```ruby
require 'victor/script`

setup width: 100, height: 100

build do
  # ...
end

save 'output.svg'
# or `puts render`
```

**Comment by DannyBen** [comment]
@gingermusketeer - I made some significant improvements. I think this PR became too large and I wish to merge it (even if you don't think it is release-ready). We can do an improvements round once we clean house a little.

Do you wish to review?

File: spec/victor-cli/ruby_code_spec.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  end
+
+  it "includes Victor::DSL methods" do
+    expect(subject).to respond_to :setup
+  end
+
+  describe "#evaluate" do
+    it "evaluate the ruby code" do
+      expect { subject.evaluate }.to output("hello\n").to_stdout
+    end
```
Comment (gingermusketeer) [inline spec/victor-cli/ruby_code_spec.rb:18]: Perhaps there should be a test here when there is a syntax error? Might be nice to make sure things like line numbers etc are still helpful.

Comment (DannyBen) [reply spec/victor-cli/ruby_code_spec.rb:18]: Right now, it shows an error without backtrace. Running with `DEBUG=1` shows the backtrace, with valid lines, but it definitely should be handler in a nicer way.

I will open an issue so one of us can take care of it post-merge.

**Pull request merged**